packages/headers-parser/vitest.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    restoreMocks: true,
+    environment: 'node',
+    onConsoleLog: (log, type) => {
+      process[type].write(log)
+    },
```

Comment:
Is this necessary? Looking at https://github.com/vitest-dev/vitest/blob/46f6bfbc9aa10ef5647005e507a8bdaddb9ad68f/packages/vitest/src/node/reporters/base.ts it should log if not set.

Reply:
By default this function returns false to discard logs: 
![CleanShot 2022-10-18 at 13 13 54](https://user-images.githubusercontent.com/11156362/196414674-fa7f7d21-8971-4d46-9e24-703157bb951b.png)

which results in not logging:
![CleanShot 2022-10-18 at 13 14 26](https://user-images.githubusercontent.com/11156362/196414756-ff25bdca-5008-4486-b65c-2d06b766d85e.png)


This is an expected behaviour to keep tests in CI clean and not log something out (but I tend to prefer having a more verbose logging output especially on debugging tests)

Reply:
I read that different, this is only the type that says either return false or void from that function.
If not set `this.ctx.config.onConsoleLog?.` will return null, which is not false.

Reply:
will remove it then